libs/atqr/domain/src/lib/domain-spec/aggegrates/Challenge Aggregate.disl

```diff
@@ -45,7 +45,7 @@ domain ATQR
           ownerPaymentMethod: PaymentMethod | null
 
           when @createdByContender
-
+            // TODO: should this function be inside ⚙️recipe ? 
```

Comment:
it seems those are rules that happen after the constructor parameters are taken, being placed inside the constructor's curly keys.